docs/04-publishing-deployment/configuration.md

```diff
@@ -21,45 +21,45 @@ The relevant sections for tuning are:
-| app.sipi.external-port                 | KNORA_WEBAPI_SIPI_EXTERNAL_PORT                 | 443                   |
-| app.ark.resolver                       | KNORA_WEBAPI_ARK_RESOLVER_URL                   | http://0.0.0.0:3336   |
-| app.ark.assigned-number                | KNORA_WEBAPI_ARK_NAAN                           | 72163                 |
-| app.salsah1.base-url                   | KNORA_WEBAPI_SALSAH1_BASE_URL                   | http://localhost:3335 |
-| app.triplestore.dbtype                 | KNORA_WEBAPI_TRIPLESTORE_DBTYPE                 | fuseki                |
-| app.triplestore.use-https              | KNORA_WEBAPI_TRIPLESTORE_USE_HTTPS              | false                 |
-| app.triplestore.host                   | KNORA_WEBAPI_TRIPLESTORE_HOST                   | localhost             |
-| app.triplestore.auto-init              | KNORA_WEBAPI_TRIPLESTORE_AUTOINIT               | false                 |
-| app.triplestore.fuseki.port            | KNORA_WEBAPI_TRIPLESTORE_FUSEKI_PORT            | 3030                  |
-| app.triplestore.fuseki.repository-name | KNORA_WEBAPI_TRIPLESTORE_FUSEKI_REPOSITORY_NAME | knora-test            |
-| app.triplestore.fuseki.username        | KNORA_WEBAPI_TRIPLESTORE_FUSEKI_USERNAME        | admin                 |
-| app.triplestore.fuseki.password        | KNORA_WEBAPI_TRIPLESTORE_FUSEKI_PASSWORD        | test                  |
-| app.cache-service.enabled              | KNORA_WEBAPI_CACHE_SERVICE_ENABLED              | true                  |
+| key in application.conf                | environment variable                            | default value           |
+| -------------------------------------- | ----------------------------------------------- | ----------------------- |
+| pekko.log-config-on-start              | KNORA_AKKA_LOG_CONFIG_ON_START                  | off                     |
+| pekko.loglevel                         | KNORA_AKKA_LOGLEVEL                             | INFO                    |
+| pekko.stdout-loglevel                  | KNORA_AKKA_STDOUT_LOGLEVEL                      | INFO                    |
+| app.print-extended-config              | KNORA_WEBAPI_PRINT_EXTENDED_CONFIG              | false                   |
+| app.bcrypt-password-strength           | KNORA_WEBAPI_BCRYPT_PASSWORD_STRENGTH           | 12                      |
+| app.jwt.secret                         | KNORA_WEBAPI_JWT_SECRET_KEY                     | super-secret-key        |
+| app.jwt.expiration                     | KNORA_WEBAPI_JWT_LONGEVITY                      | 30 days                 |
+| app.jwt.issuer                         | KNORA_WEBAPI_JWT_ISSUER                         | 0.0.0.0:3333            |
+| app.dsp-ingest.audience                | KNORA_WEBAPI_DSP_INGEST_AUDIENCE                | <http://localhost:3340> |
+| app.dsp-ingest.base-url                | KNORA_WEBAPI_DSP_INGEST_BASE_URL                | <http://localhost:3340> |
```

Comment:
I intuitively don't like the pointy brackets around the URLs... is that standard?

Reply:
The drawback of markdown is that it is not really standardized. These angle brackets have been introduced by `markdownlint-cli2 --fix`, so I guess we can call it standardized.